content/en/borealis/not-a-real-page.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Fake page to watch changes
+linkTitle: 
+weight: 99999
+summary: "This is not a real page. It's a draft file I use to update include files so that they render in builds dynamically."
+draft: true
+---
+
+Each Kubernetes cluster that you install the RNA on should have a unique name that you set using the `agentIdentifier` parameter. This identifier is used in the Status UI and in other places to identify the cluster. Note that older versions of the RNA used the `agent-k8s.accountName` parameter for this.
+
+1. Verify that you are in the correct Kubernetes context. The RNA is installed in the target deployment cluster.
+
+2. Add the Armory Helm repo:
+
+   ```bash
+   helm repo add armory https://armory.jfrog.io/artifactory/charts
+   ```
+
+   You only need to do this once.
+
+3. Refresh the repo cache:
+
+   ```bash
+   helm repo update
+   ```
+
+4. Install the Helm chart. Keep the following in mind when you install the RNA:
+
+   * You can create the `armory-rna` namespace before running the `helm upgrade --install` command or include the `--create-namespace` option.
+
+   * The `agentIdentifier` option is the name that is used to refer to the deployment target cluster in the Status UI, your deployment file, and other places, so use a descriptive name.
+
+   * Armory recommends storing and using the values for`clientID` and `clientSecret` as secrets. You can use either a secret engine directly or environment variables to store and pass the encrypted values.
+
+     * If you use a secret engine, pass the encrypted value for the parameter in the command, such as `'encrypted:k8s!n:rna-client-credentials!k:client-secret'`. Armory supports the following secret engines: Vault, Kubernetes secrets, encrypted GCS buckets, encrypted S3 buckets, and AWS Secrets Manager. See the documentation for your secrets engine for the format of the encrypted value.
```

Comment:
Can we add the recommended k8s example and hide some of these details

```shell
kubectl --namespace armory-rna create secret generic rna-client-credentials --type=string --from-literal=client-secret=xxx-yyy-ooo --from-literal=client-id=zzz-ooo-qqq
```

Reply:
We might just nix this whole block about secrets and just add using k8s secrets as a step and have a quick comment about there being more there or something

Reply:
yes, let's please be prescriptive about this. Really I want the quick starts to be step by step instructions with no options.